demos/src/scss/_demos.scss

```diff
@@ -48,10 +48,9 @@ body:before {
 }
 
 .o-grid-container {
-	background: lighten(#f6e9d8, 8.5%);
+	background: #EAE8DF;
 }
```

Comment:
Has anyone ever said it looks like a bug? I think it's useful to show these 'seams' - aids comprehension and, as kaelig says,  a useful debugging tool